src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariantsIntegrationTest.java

```diff
@@ -155,6 +156,9 @@ public void testOnContigEdge() throws IOException {
      */
     @Test(groups = {"python"})
     public void testInference2dResourceModel() throws IOException {
+        // We reset the random number generator at the beginning of each test so that the random down-sampling of reads
+        // by the reservoir down-sampler does not cause slightly different scores.
+        Utils.getRandomGenerator().setSeed(1234);
```

Comment:
Call `Utils.resetRandomGenerator()` instead (here and below)

Reply:
got it!